### PR TITLE
Add alt text to cards

### LINKED ISSUE #3927: [UWP][Accessibility] Name attribute is not defined for 'Spider' image under 'Solitaire.JSON' list item.
# Platform
* UWP

# Version of SDK
master

# Details

Repro Steps:​
1. Launch the application.​
2. New screen starts appearing. Navigate to Left pane JSOn files.​
3. Now select "Solitaire.JSON" button. It will update the Middle and Right Pane. Navigate all the elements of both the panes.​
4. Verify alt of 'Spider' image control by using accessibility insights for desktop.

Actual Result:
Name attribute is not defined for 'Spider' image under 'Solitaire.JSON' list item and screen reader is reading as image while navigating in scan mode, due to this user will not get the purpose of image control which presents in right pane.

Expected Result:
Name attribute should be define for 'Spider' image under 'Solitaire.JSON' list item and screen reader should be read as 'Spider' image while navigating in scan mode, so that user can easily understand the purpose of image control which presents in right pane.

### LINKED ISSUE #3929: [UWP][Accessibility] Name attribute is not defined for image under 'SportingEvent.JSON' right pane.
# Platform
* UWP

# Version of SDK
master

# Details
Repro Steps:​
1. Launch the application.​
2. New screen starts appearing. Navigate to Left pane JSOn files.​
3. Now select "SportingEvent.JSON" button. It will update the Middle and Right Pane. Navigate all the elements of both the panes.
4. Verify alt of 'Shades and skins' image control by using accessibility insights for desktop.

Actual Result:
Name attribute is not defined for 'Shades and skins' image under 'SportingEvent.JSON' right pane and screen reader is reading as image while navigating in scan mode, due to this user will not get the purpose of image control which presents in right pane.

Expected Result:
Name attribute should be define for 'Shades and skins' image under 'SportingEvent.JSON' right pane, so that user can easily understand the purpose of image control which presents in right pane.

### LINKED ISSUE #3931: [UWP][Accessibility] Name attribute is not defined for image under Agenda.JSON file.
# Platform
* UWP

# Version of SDK
master

# Details
Repro Steps:​
1. Launch the application.
2. New screen starts appearing. Navigate to Left pane JSOn files.​
3. Now select "Agenda.JSON" button. It will update the Middle and Right Pane. 
4. Set to scan mode by using Caps + Space keys.
5. Now, navigate to image section at right pane.
6. Verify alt of image.

Actual Result:
Name attribute is not defined for image appearing at right pane of Agenda.JSON file and due to this, screen reader user is unable to get the purpose of image control.

Expected Result:
There must be name attribute defined for Image control appearing at right pane of Agenda.JSON file so that screen reader user could get the context of image control.

### PR DESCRIPTION
**Pull request by almedina-ms**
## Related Issue
Fixes #3927
Fixes #3929
Fixes #3931

## Description
Adds altText property to images in sample files to provide more context to users of screen readers while navigating the adaptive cards.

## How Verified
The fix was verified using the accessibility insights application on the visualizer to verify that the name property has been filled in

image

image

image

image

image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/3928)

### CODE REVIEW

File: samples/v1.2/Scenarios/Agenda.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
 									"items": [
 										{
 											"type": "Image",
-											"url": "https://messagecardplayground.azurewebsites.net/assets/LocationGreen_A.png"
+											"url": "https://messagecardplayground.azurewebsites.net/assets/LocationGreen_A.png",
+											"altText": "Stop A"
```
Comment (shalinijoshi19) [inline samples/v1.2/Scenarios/Agenda.json:21]: >Stop [](start = 23, length = 4)

Should we match this to reflect the image name vis LocationGreen_A ?

File: samples/v1.2/Scenarios/Agenda.json [hunk 2]
```diff
@@ -175,7 +179,8 @@
 						{
 							"type": "Image",
 							"horizontalAlignment": "Center",
-							"url": "https://messagecardplayground.azurewebsites.net/assets/CircleGreen_coffee.png"
+							"url": "https://messagecardplayground.azurewebsites.net/assets/CircleGreen_coffee.png",
+							"altText": "Coffee icon"
```
Comment (shalinijoshi19) [inline samples/v1.2/Scenarios/Agenda.json:183]: >Coffee icon" [](start = 19, length = 12)

Is "icon" redundant here? Alternatively should the other alttext strings include qualifiers like "icon" or "image" too? What's guidance say here?

File: samples/v1.2/Scenarios/Agenda.json [hunk 3]
```diff
@@ -197,7 +202,8 @@
 									"items": [
 										{
 											"type": "Image",
-											"url": "https://messagecardplayground.azurewebsites.net/assets/location_gray.png"
+											"url": "https://messagecardplayground.azurewebsites.net/assets/location_gray.png",
+											"altText": "Location"
```
Comment (shalinijoshi19) [inline samples/v1.2/Scenarios/Agenda.json:206]: Location_gray?

File: samples/v1.2/Scenarios/Agenda.json [hunk 4]
```diff
@@ -380,7 +392,8 @@
...
```
Comment (shalinijoshi19) [inline samples/v1.2/Scenarios/Agenda.json:396]: LocationGray?

**Comment by shalinijoshi19** [comment]
@dclaux is added to the review. #Closed

File: samples/v1.2/Scenarios/Agenda.json (hunk 2 above)
Comment (shalinijoshi19) [reply samples/v1.2/Scenarios/Agenda.json:183]: @almedina-ms here's what Sarah from Accessibility team pointed to : https://ux.iu.edu/accessibility/alt-text/ and the Msft training: https://aka.ms/AltTextClass

File: samples/v1.2/Scenarios/Agenda.json (hunk 1 above)
Comment (almedina-ms) [reply samples/v1.2/Scenarios/Agenda.json:21]: I'd think that we can keep it simple as Location A. I don't know if we should use color names in alt text but I would guess we shouldn't

File: samples/v1.2/Scenarios/Agenda.json [hunk 5]
```diff
@@ -137,7 +140,8 @@
 										{
 											"type": "Image",
 											"horizontalAlignment": "Left",
-											"url": "https://messagecardplayground.azurewebsites.net/assets/Conflict.png"
+											"url": "https://messagecardplayground.azurewebsites.net/assets/Conflict.png",
+											"altText": "Conflict"
```
Comment (shalinijoshi19) [inline samples/v1.2/Scenarios/Agenda.json:144]: Maybe slightly more descriptive? "Source Conflict icon" or something?  Btw I wonder if these URL's need to be updated at all? The raw URL takes me to the messagecardsplayground landing page not hte image itself ?

**Pull request merged**